python/cli/prelude_cli/views/configure.py

```diff
@@ -1,13 +1,47 @@
 import click
 
+from configparser import ConfigParser, MissingSectionHeaderError
+
 
 @click.command()
+@click.option('-f', '--file', help="path to credentials (.ini) file", default=None, type=click.Path(exists=True))
 @click.pass_obj
-def configure(account):
+def configure(account, file):
     """ Configure your local keychain """
-    profile = click.prompt('Enter the profile name', default=account.profile, show_default=True)
-    hq = click.prompt('Enter the Prelude API', default=account.hq, show_default=True)
-    account_id = click.prompt('Enter your account ID')
-    account_token = click.prompt('Enter your account token')
-    account.configure(account_id=account_id, token=account_token, hq=hq, profile=profile)
-    click.secho('Credentials saved', fg='green')
+    profile = account.profile
+    hq = account.hq
+    account_id = None
+    account_token = None
+
+    if file:
+        config = ConfigParser()
+
+        try:
+            config.read(file)
+            profile = config.sections()[0]
+            values = config[profile]
```

Comment:
So it looks like we're taking the first entry from the file... what about that case where a keychain has multiple entries? Should we instead grab the "default" section and override the account.profile section? Or should that be configurable via additional flag? Thoughts?

Reply:
Actually, the `Account` object has wrappers to handle the reading and importing of a keychain... 

Maybe we should refactor the Account class in Account.py to allow us to read in a keychain that's not the default keychain.

Here's the current function:
```python
def read_keychain_config(self, hq='https://api.preludesecurity.com', profile='default'):
        if not exists(self.keychain_location):
            head, _ = os.path.split(Path(self.keychain_location))
            Path(head).mkdir(parents=True, exist_ok=True)
            open(self.keychain_location, 'x').close()
            self.configure('', '', hq, profile)
        cfg = configparser.ConfigParser()
        cfg.read(self.keychain_location)
        return cfg
```

I think we can probably clean up this function to do something like:
```
try:
      valid_config = account.read_keychain_config(file_path=file, profile='default')
      account.configure(**valid_config)
 except:
     whoops bad config
```